src/zkevm_specs/evm/execution/call_staticcall.py

```diff
@@ -173,7 +197,11 @@ def call(instruction: Instruction):
             (CallContextFieldTag.CallDataLength, cd_length),
             (CallContextFieldTag.ReturnDataOffset, rd_offset),
             (CallContextFieldTag.ReturnDataLength, rd_length),
-            (CallContextFieldTag.Value, value.expr()),
+        ]
+        # Value is not used for opcode STATICCALL.
+        if is_call == 1:
+            call_context_lookups.append((CallContextFieldTag.Value, value.expr()))
```

Comment:
We still need to initialize the `CallContextFieldTag.Value` to `0` in for next call when it's `STATICCALL`, otherwise a malicious prover can put any value there.

Reply:
Fixed to initialize `CallContextFieldTag.Value` to `0` for opcode `STATICCALL`.